strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/storage/repository/remote/heartbeat/monitor/HttpGetRemoteRepositoryCheckStrategy.java

```diff
@@ -24,15 +25,15 @@
     private static final Logger logger = LoggerFactory.getLogger(HttpGetRemoteRepositoryCheckStrategy.class);
 
     @Inject
-    private ProxyRepositoryConnectionPoolConfigurationService proxyRepositoryConnectionPoolConfigurationService;
+    private ProxyRepositoryConnectionConfigurationService proxyRepositoryConnectionConfigurationService;
```

Comment:
Just for my better understanding: why is this no lnger using the connection pool service configuration?

Reply:
`ProxyRepositoryConnectionConfigurationService` is a wrapper over `ProxyRepositoryConnectionPoolConfigurationService`, so both are doing same thing.